src/lib/types.ts

```diff
@@ -87,6 +87,17 @@ export type Lang =
   | "zh"
   | "zh-tw"
 
+export type Direction = "rtl" | "ltr" | "auto"
+
+export type Language = {
```

Comment:
Maybe we can start calling this something different now, what do you think?

We have started using `locale`. Perhaps we can call this `ConfigLocale` or `LocaleConfig` since this structure is coming from the i18n config file.

Reply:
Yeah I'm fine with that... `locale` is probably best given it's the terminology already used by `useRouter()`

Reply:
How about `I18nLocale`? Using `Config` makes me feel like it should be the containing array of locale items (like the whole configuration JSON)

Reply:
That is much better! agreed, mine was a horrible name haha